File: php/testcode/benchmark_test_01022.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest01022(BenchmarkRequest $req): BenchmarkResponse {
    $key = random_bytes(16);
    $enc = openssl_encrypt($req->param('data'), 'RC4', $key);
    return BenchmarkResponse::ok($enc);
}

File: rust/testcode/benchmark_test_01022.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let token = req.param("admin_token");
    let cookie = format!("admin={}; Path=/admin; Secure; HttpOnly; SameSite=Strict", token);
    super::shared::BenchmarkResponse::ok(&format!("Set-Cookie: {}", cookie))
}
